libtransmission/torrent.h

```diff
@@ -240,6 +241,8 @@ struct tr_torrent
     uint16_t idleLimitMinutes;
     tr_idlelimit idleLimitMode;
     bool finishedSeedingByIdle;
+
+    tr_ptrArray labels;
```

Comment:
This is more of a question than requirement... but in practice in other clients, do lots of torrents wind up using the same handful of labels?

I'm wondering if it would make sense to make these quarks instead of strings, so that we don't have hundreds of copies of the same string

Reply:
Since this is completely up to the user I imagine it varies a lot.
For my use case I expect to have hundreds of common labels and also hundreds of individual ones.
For example many torrents can be labeled as "sci-fi" and "books" and many will have individual labels for author, year, whatever.